src/main/java/reposense/system/CommandRunner.java

```diff
@@ -96,10 +96,10 @@ public static String getCommitHashBeforeDate(String root, String branchName, Dat
         return runCommand(rootPath, revListCommand);
     }
 
-    public static String cloneRepo(String org, String repoName) throws IOException {
-        Path rootPath = Paths.get(Constants.REPOS_ADDRESS, org);
+    public static String cloneRepo(String location, String displayName) throws IOException {
+        Path rootPath = Paths.get(Constants.REPOS_ADDRESS, displayName);
```

Comment:
Let's have here use `getRepoDirectory`, shift `REPOS_ADDRESS` to `FileUtil.java` and destroy the `Constants` class.